core/model/src/main/java/com/yapp/model/exceptions/YappExceptions.kt

```
@@ -0,0 +1,24 @@
+package com.yapp.model.exceptions
+
+enum class YappServerError(val exception: YappException) {
+    USR_1001(SignUpCodeException()),
+    USR_1002(AlreadyRegisteredException()),
+    USR_1003(UnprocessedSignUpException()),
+}
+
+class SignUpCodeException() : YappException("잘못된 가입코드입니다.")
```

detekt (1.23.7)

[warning] An empty default constructor can be removed.

(detekt.empty-blocks.EmptyDefaultConstructor)